server.js

```diff
@@ -8,9 +8,11 @@ var _ = require('lodash');
 var routeHandler = require('./src/routeHandler');
 
 var PORT = 8000;
+var VAULT_URL_DEFAULT = process.env.VAULT_URL_DEFAULT || "";
 
 var app = express();
-app.set('view engine', '.html');
+app.set('view engine', 'html');
```

Comment:
handlebars added a simple templating language.  This was necessary to pass data along with the html file.